server/form-pages/apply/risk-and-need-factors/prison-information/caseNotes.ts

```diff
@@ -58,7 +59,10 @@ export const caseNoteCheckbox = (caseNote: PrisonCaseNote, checked: boolean) =>
   `
 }
 
-@Page({ name: 'case-notes', bodyProperties: ['caseNoteIds', 'selectedCaseNotes', 'moreDetail', 'adjudications'] })
+@Page({
+  name: 'case-notes',
+  bodyProperties: ['caseNoteIds', 'selectedCaseNotes', 'moreDetail', 'adjudications', 'acctAlerts'],
```

Comment:
Think this means the schema will need updating 👍

Reply:
Correct :)